src/matlab2tikz.m

```diff
@@ -3099,16 +3103,14 @@ function assertRegularAxes(handle)
     % Not that, in Pgfplots, long texts get cut off at the axes. This is
     % Different from the default MATLAB behavior. To fix this, one could use
     % /pgfplots/after end axis/.code.
-
-    str = '';
-
+    
     % - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
     % get required properties
-    String = get(handle, 'String');
+    textString = get(handle, 'String');
```

Comment:
Somehow, I don't like the name `textString` (not that `String` was any better, though), but I think it is somewhat pleonastic: text is a string (or a string is text).

How about `contents`? Or `label`? Or just `text`?